### PR TITLE
Update Json.NET to 9.0.1-beta1

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
       "type": "build"
     },
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "9.0.1-beta1",
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json]: Can we remove `imports` from this project if we update to this version?

Comment (javiercn) [inline src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json]: I'll do it on a separate commit

Comment (dougbu) [inline src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json]: @javiercn a separate commit or a separate PR? Thought removing the `imports` was why we were doing this version change or at least one of the major points.

**Comment by javiercn** [comment]
I mean a different PR, it's way safer to first update all repositories to use 9.0.1-beta1 and then, after a successful build remove all the unnecessary imports from the repositories. Doing it this way, allows me to update repositories one by one without running into restore problems locally
